### DISCUSSION TITLE
Increased "perPage=" param in GET requests

### DISCUSSION THREAD

**Discussion by haraldroine** [pocketbase/pocketbase#3451] [Q&A]
Hello, 
I am running a high record volume self-hosted version of PocketBase. 
In the instance I'm running, I need to fetch multiple 10.000s per GET request. But I stumbled upon a limit of 500 records per page with the GET /api/collections/COLLECTIONNAMEHERE/records. 

I'm using the ?perPage=10000, param in the URL. 

However, I'm only getting 500 records per page in the results back.

Any idea on how I can increase this?
If possible?

Thanks.
-Harald

**Comment by ganigeorgiev** [comment]
The limit is a fixed constant and it is not possible to change at the moment.
